pip/basecommand.py

```diff
@@ -214,10 +215,20 @@ def main(self, args):
         if options.require_venv:
             # If a venv is required check if it can really be found
             if not running_under_virtualenv():
-                logger.critical(
-                    'Could not find an activated virtualenv (required).'
-                )
-                sys.exit(VIRTUALENV_NOT_FOUND)
+                if not getattr(options, 'use_user_site'):
```

Comment:
this is odd to me. If I've set the option to require a virtualenv, it shouldn't allow a user install?  that's redefining the option.